server/src/test/java/org/elasticsearch/runtimefields/mapper/DoubleScriptFieldTypeTests.java

```diff
@@ -259,7 +259,7 @@ public String getType() {
 
                     @Override
                     public Set<ScriptContext<?>> getSupportedContexts() {
-                        return Set.of(DoubleFieldScript.CONTEXT);
+                        return Set.of();
```

Comment:
I'm kind of surprised this passes without this.

Reply:
I poked a bit with this and I think this is because all the runtime fields contexts are now built-in and applied by ScriptModule by default.

Reply:
Weird! I thought the scripting infra wouldn't send a `Script` to an engine that didn't support the context. But it looks like that isn't how it works. It more like script engines can make new contexts with this thing.